ts/defs.bzl

```diff
@@ -83,29 +83,19 @@ def ts_project(
     Unlike bare `tsc`, this rule understands the Bazel interop mechanism (Providers)
     so that this rule works with others that produce or consume TypeScript typings (`.d.ts` files).
 
-    One of the benefits of using ts_project is that it understands Bazel Worker Protocol which makes
-    JIT overhead one time cost. Worker mode is on by default to speed up build and typechecking process.
+    One of the benefits of using ts_project is that it understands the [Bazel Worker Protocol]
```

Comment:
Was "Bazel Worker Protocol" meant to be a link?

Reply:
it is, using the "reference-style links" markdown syntax  https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
TIL... 👍